docs/stac-spec/extensions/layer/json-schema/item-schema.json

```diff
@@ -0,0 +1,47 @@
+  
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "item-schema.json#",
+    "title": "Layer Extension",
+    "description": "STAC Layer Item Extension Spec",
+    "allOf": [
+        {
+            "$ref": "https://raw.githubusercontent.com/radiantearth/stac-spec/dev/item-spec/json-schema/item.json"
+        },
+        {
+            "$ref": "#/definitions/layer"
+        }
+    ],
+    "definitions": {
+        "layer": {
+            "type": "object",
+            "required": [
+                "stac_extensions",
+                "properties"
+            ],
+            "properties": {
+                "stac_extensions": {
+                    "type": "array",
+                    "contains": {
+                        "const": "layer"
+                    }
+                },
+                "properties": {
+                    "type": "object",
+                    "required": [
+                        "layer:ids"
+                    ],
+                    "properties": {
+                        "layer:ids": {
+                            "title": "Layer Names",
+                            "type": "array",
+                            "items": {
+                                "type": "string"
```

Comment:
Do you think it would be valuable to require these strings to be non-empty? I don't know how to make a url with an empty string component

Reply:
@jisantuc I copied it from the label extension. Yea, I wanted to admit that if the datetime would be specified it should follow the STAC datetime rules.

Reply:
Gonna collapse this and we can talk about it on the previous comment